operator/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang as builder
+LABEL maintainer="maintainer@cilium.io"
+WORKDIR /go/src/github.com/cilium/cilium
+COPY . ./
+ARG LOCKDEBUG
+ARG V
+RUN make -C operator LOCKDEBUG=$LOCKDEBUG V=$V
+
+FROM ubuntu:18.04
```

Comment:
Would be amazing if it would work with: 
```
CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' make -C operator
....
FROM scratch
COPY --from=builder /go/src/github.com/cilium/cilium/operator/cilium-operator /usr/bin/cilium-operator
CMD ["/usr/bin/cilium-operator"]
```

Reply:
A bit more work is needed to remove the bpf golang dependencies from pkg/endpoint and pkg/k8s. We are on the way there.